app/controllers/charges_controller.rb

```diff
@@ -24,31 +25,57 @@ def create
       seller_names: seller_names.to_a,
       item_types: item_types
     )
-    intent = Stripe::PaymentIntent.create(
-      amount: amount,
-      currency: 'usd',
-      receipt_email: charge_params[:email],
-      payment_method_types: ['card'],
-      description: description
-    )
 
-    # Creates a pending PaymentIntent. See webhooks_controller to see what happens
-    # when the PaymentIntent is successful.
-    PaymentIntent.create!(
-      stripe_id: intent['id'],
-      email: charge_params[:email],
-      line_items: line_items.to_json
-    )
-
-    json_response(intent)
+    if ENV['use_square']
+      api_client = Square::Client.new(
+        # TODO: put in ENV
+        access_token: 'EAAAEAQN-8I264d9ntPRq8iqR2I9jY94VEmdngHcGAghHSdAXjdFVcp7-DfECWXI',
+        # environment: ENV['IS_PRODUCTION'] == 'false' ? 'sandbox' : 'production'
```

Comment:
everything in the .env file is test/dev. production variables are stored on heroku